### PR TITLE
Feature/cvsb 416

### PR DESCRIPTION
**Pull request by oalexandru**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# cvs-svc-technical-records
```
Comment (gmlazu) [inline README.md]: test-results

File: README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+### Environmental variables
+
+- The `BRANCH` environment variable indicates in which environment is this application running. Use `BRANCH=local` for local deployment. This variable is required when starting the application or running tests.
+
+
+### Seeding the database
+
+- From within the resource folder type the following:
+```cat techRecords.json | json-dynamo-putrequest cvs-BRANCH-dft-tech-records --beautify >test.json```
```
Comment (gmlazu) [inline README.md]: I think we can get rid of this. Seeding can be done via serverless.

File: tests/integration/testResults.intTest.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            })
+          })
+        })
+
+        context('and there are no test results for that VIN that have status \'cancelled\'', () => {
+          it('should return 404', (done) => {
+            request.get('test-results/1B7GG36N12S678425?status=cancelled')
+              .end((err, res) => {
+                if (err) { expect.fail(err) }
+                console.log(res.body)
```
Comment (gmlazu) [inline tests/integration/testResults.intTest.js]: Do we need this logging?

File: README.md (hunk 1 above)
Comment (oalexandru) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (oalexandru) [reply README.md]: done

File: tests/integration/testResults.intTest.js (hunk 3 above)
Comment (oalexandru) [reply tests/integration/testResults.intTest.js]: nope. done

**Pull request merged**